core/src/main/java/io/grpc/internal/AbstractServerStream.java

```diff
@@ -177,6 +177,11 @@ public StatsTraceContext statsTraceContext() {
     return statsTraceCtx;
   }
 
+  @Override
+  public void setOnReadyThreshold(int numBytes) {
+    transportState().setOnReadyThreshold(numBytes);
```

Comment:
This is only here to make it public? Then:
```suggestion
    super.setOnReadyThreshold(numBytes);
```